react-common/components/controls/EmbedVideo.tsx

```diff
@@ -0,0 +1,52 @@
+import * as React from "react";
+import { classList, ControlProps } from "../util";
+
+export interface EmbedVideoProps extends ControlProps {
+  src: "youtube" | "streams";
+  id: string;
+  autoplay?: boolean;
+  allowFullScreen?: boolean;
+//   encryptedmedia?: boolean;
+
+}
+
+
+
+export const EmbedVideo = (props: EmbedVideoProps) => {
+    const {
+       src,
+       id,
+       autoplay,
+       allowFullScreen
+
+    } = props;
+
+
+
+    let videoURL = `https://www.youtube.com/embed/${id}?${autoplay ? "autoplay=1" : ""}`;
+    let streamURL= `https://msit.microsoftstream.com/video/${id}`;
```

Comment:
the root path should probably be configurable in pxtarget.json (this can be the default, but a setting can override it)

Reply:
sounds good, thank for feedback! will implement partially, but make more full use once a little farther